extensions/amp-render/1.0/amp-render.js

```diff
@@ -168,6 +168,10 @@ export class AmpRender extends BaseElement {
     this.initialSrc_ = this.element.getAttribute('src');
     this.src_ = this.initialSrc_;
 
+    if (!this.element.hasAttribute('aria-live')) {
+      this.element.setAttribute('aria-live', 'polite');
+    }
```

Comment:
Another question - does this have to be on the custom element? Can it be done in the Preact layer?

Reply:
If this attribute is added to the Preact layer, any changes to placeholder & fallback will not be announced by screen readers.

Reply:
I see. What about changes within the Preact component, such as actual data rendering into a template? I assume that will be announced by AMP because of this logic but not in Preact? If we add the attribute in Preact layer also, will it cause redundant announcements? If not, perhaps both locations is a good compromise.

Reply:
Yes, with the current logic any changes to the content including placeholder, fallback and rendered data will be announced. By moving it just to the Preact layer, it skips announcing fallback and placeholder updates.

I added the attribute to the Preact layer too and based on testing with VoiceOver on Mac, there are no redundant announcements. Guess that's a good compromise 🙂